docs/cloud-provider-specifics.md

```diff
@@ -1,5 +1,7 @@
 # Cloud Provider Specifics
 
+> NOTE: Documentation may change between releases. See the [Changelog][20] for links to previous versions of this repository and its docs.
```

Comment:
I'm a little concerned since I don't know how many people will do the quickstart (as opposed to cloning and jumping straight to these instructions). Both this file and the quickstart have code examples that readers are meant to actually go through, which can be inconsistent with the latest release.

I'd rather err on the side of being a bit redundant (if it saves people the frustration of debugging/finding the note in the main README), but if you feel strongly, I can take this out.